honeybee_3dm/model.py

```diff
@@ -29,6 +29,8 @@ def import_3dm(path, name=None):
         path: A text string for the path to the rhino3dm file.
         name: A text string that will be used as the name of the Honeybee
             model. Default will be the same as Rhino file name.
+        visibility: Bool. If set to False then the objects on an "off"
```

Comment:
I would change this keyword to something more explicit. What about `import_on_layers`?

Visibility can be confusing since you might have a hidden object in a layer with the visibility set to on.

Reply:
Hi @mostaphaRoudsari,
I looked at rhino API and found this term [visibility](https://developer.rhino3d.com/wip/api/RhinoCommon/html/M_Rhino_DocObjects_Layer_GetPersistentVisibility.htm) in the context of layers, so I went ahead with it. If you still want to change it can we do `layer_visibility` ?

Reply:
I like the `layer_visibility` option.